packages/react-performance/src/test/LifecycleEventListener.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
```

Comment:
I'm not sure I agree, I think generally we have avoided mocking hooks in tests.  Including `context` here is nice because it makes the test closer to how you would actually expect to use the component. Tests as documentation is very useful IMO.